tests/e2e/support/environment/actor/actor.ts

```diff
@@ -28,6 +28,14 @@ export class ActorEnvironment extends EventEmitter implements Actor {
     this.page = await this.context.newPage()
   }
 
+  public async newPage(newPage: Page): Promise<Page> {
+    // close the old page
+    await this.page.close()
+    // set the new page
+    this.page = newPage
+    return this.page
+  }
```

Comment:
Here, actor's `page` will be replaced with `new tab` and the `old tab` is `closed`
Please, suggest if it feels odd and if you have other great ideas.

Reply:
if we can store browser context for multiple pages then it will be helpful in the future when we need to open a new page do some tasks and come back to the old page and do some task and close the old page.
Or it is not needed for now?

Reply:
IMHO may be needed in the future, but not in this PR